interfaces.go

```diff
@@ -134,8 +135,8 @@ type ContractCaller interface {
 // FilterQuery contains options for contract log filtering.
 type FilterQuery struct {
 	BlockHash *common.Hash     // used by eth_getLogs, return logs only from block with this hash
-	FromBlock *big.Int         // beginning of the queried range, nil means genesis block
-	ToBlock   *big.Int         // end of the range, nil means latest block
+	FromBlock *rpc.BlockNumber // beginning of the queried range, nil means genesis block
```

Comment:
If you are defining `FromBlock` and `ToBlock` to be of these types, perhaps you don't need `UnmarshalJSON` function for `FilterQuery` type anymore? My understanding is that it only existed to convert `*rpc.BlockNumber` to `*big.Int`

Reply:
I still need UnmarshalJSON for `Account` that can be either a single value or a slice

Reply:
I've reverted this type change